geode-redis/src/main/java/org/apache/geode/redis/internal/netty/Command.java

```diff
@@ -183,8 +184,16 @@ public boolean isOfType(RedisCommandType type) {
   }
 
   public String wrongNumberOfArgumentsError() {
-    return String.format("wrong number of arguments for '%s' command",
-        getCommandType().toString().toLowerCase());
+    String result;
+
+    if (getCommandType().equals(RedisCommandType.INFO)) {
```

Comment:
Maybe a nit, but it feels like this is now hidden in a class that is unrelated to anything about parameters. I much preferred your previous idea about passing the 'special' error message in at the constraint level - i.e. in `RedisCommandType`. Perhaps too much refactoring, but also consider making `ParameterRequirements` abstract and pulling this method into that class. It feels like a more natural place for it.

Reply:
yeah, we discussed that same thing.  I think we both saw this point but decided not to make the change because we had just finished doing it the other way.  Anyway, just pushed a commit that implements it the way we originally discussed (with the parameter being passed in when the command is declared in the RedisCommandTypes file).